internal/receiver/scriptreceiver/README.md

```diff
@@ -0,0 +1,84 @@
+# Script Receiver
+
+<!-- status autogenerated section -->
+| Status        |                        |
+| ------------- |------------------------|
+| Stability     | [in-development]: logs |
+| Distributions | [contrib]              |
+
+[in-development]: https://github.com/open-telemetry/opentelemetry-collector#in-development
```

Comment:
```suggestion
[in-development]: https://github.com/open-telemetry/opentelemetry-collector#development
```

Reply:
I think this is the final item and I'll be sure to merge when updated.